kratos/python_scripts/vtk_output_process.py

```diff
@@ -37,11 +37,11 @@ def __init__(self, model, settings ):
         if self.settings["save_output_files_in_folder"].GetBool():
             if self.model_part.GetCommunicator().MyPID() == 0:
                 folder_name = self.settings["folder_name"].GetString()
-                if os.path.isdir(folder_name) and not \
-                    self.model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED]:
-                    import shutil
-                    shutil.rmtree(folder_name) # removing old results
-                os.mkdir(folder_name)
+                if not self.model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED]:
+                    import KratosMultiphysics.kratos_utilities as kratos_utils
+                    kratos_utils.DeleteDirectoryIfExisting(folder_name)
```

Comment:
Does this delete even if the dir is not empty ?

Reply:
yes, does the exact same as before, just using an already existing function